pkgs/development/tools/picotool/default.nix

```
@@ -15,6 +15,11 @@ stdenv.mkDerivation rec {
   nativeBuildInputs = [ cmake pkg-config ];
   cmakeFlags = [ "-DPICO_SDK_PATH=${pico-sdk}/lib/pico-sdk" ];
 
+  postInstall = ''
+    mkdir -p $out/etc/udev/rules.d
+    cp ../udev/99-picotool.rules $out/etc/udev/rules.d
```

```suggestion
    install -Dm444 ../udev/99-picotool.rules -t $out/etc/udev/rules.d
```

and the `mkdir` line can be removed